Rust/Rust.sublime-syntax

```diff
@@ -238,9 +238,10 @@ contexts:
         1: punctuation.definition.annotation.rust
         2: punctuation.section.group.begin.rust
       push:
+        - meta_scope: meta.annotation.rust
```

Comment:
Doesn't this give a double meta.annotation scope? See line 236 above here. If that's the case, maybe use meta_content_scope?